src/App.tsx

```diff
@@ -16,46 +16,106 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [isReverse, setIsReverse] = useState(false);
+  const [sortType, setSortType] = useState('NONE');
+
+  const alphabet = () => {
```

Comment:
Give more clear names for these functions. To make it clear that these are actions

Reply:
not fixed